data/meson.build

```diff
@@ -3,11 +3,11 @@
 # Installing the public DBus API
 install_data(
     [
-        'org.containers.hirte.Job.xml',
-        'org.containers.hirte.Manager.xml',
-        'org.containers.hirte.Monitor.xml',
-        'org.containers.hirte.Node.xml',
-        'org.containers.hirte.Agent.xml'
+        'org.containers.bluechi.Job.xml',
+        'org.containers.bluechi.Manager.xml',
+        'org.containers.bluechi.Monitor.xml',
+        'org.containers.bluechi.Node.xml',
+        'org.containers.bluechi.Agent.xml'
```

Comment:
org.containers.hirte. should be replaced with io.github.eclipse-bluechi.bluechi according to the doc and today's discussion